### ISSUE TITLE
Add already existing posts to blog

### ISSUE THREAD

**Issue by Anth0rx** [wise-team/engrave#96]
Currently I am missing the feature to add already existing posts to a newly created blog. It is only possible to add a new post.

**Comment by bgornicki** [comment]
It is possible to import old posts when creating a new blog.

**Comment by Anth0rx** [comment]
Okay, I have found it. Is is also possible to import posts after a blog is already created?

**Comment by bgornicki** [comment]
No, it's possible only while creating a new blog and it possible to do once.
